app.js

```diff
@@ -2,15 +2,36 @@ const app = Vue.createApp({
     data() {
         return {
             cvssConfigData: cvssConfig,
-            cvssLookupData: cvssLookup,
+            maxComposedData: maxComposed,
+            maxHammingData: maxHamming,
             cvssMacroVectorDetailsData: cvssMacroVectorDetails,
             cvssMacroVectorValuesData: cvssMacroVectorValues,
             showDetails: false,
             cvssSelected: null,
-            header_height: 0
+            header_height: 0,
+            enviro_default: "H",
+            currentLookup: cvssLookup_global,
         }
     },
     methods: {
+        getvalueEqLookup(lookup,i){ 
+            eq=parseInt(lookup[i])
+            eq_val = maxComposed["eq"+String(i+1)][eq]
+            return eq_val
+        },
+        extractValueMetric(metric,str){
```

Comment:
Please follow previously used code style consistently. Especially:

- Use of white-spaces around operators, around commas, before curly braces, in the beginning of comments etc.
- Variable naming conventions, use camelCase as is customary in Javascript code.
- Comments starting with capital letter, if it is a complete sentence, end with period.